optuna/visualization/_utils.py

```diff
@@ -26,6 +32,29 @@ def is_available() -> bool:
     return _plotly_imports._imports.is_successful()
 
 
+def _check_plot_args(
+    study: Union[Study, Sequence[Study]],
+    target: Optional[Callable[[FrozenTrial], float]],
+    target_name: str,
+) -> None:
+    if isinstance(study, Study):
+        studies = [study]
+    else:
+        studies = study
+
+    if target is None and any(study._is_multi_objective() for study in studies):
+        raise ValueError(
+            "If the `study` is being used for multi-objective optimization, "
+            "please specify the `target`."
+        )
+
+    if target is not None and target_name == "Objective Value":
+        warnings.warn(
+            "`target` is specified, but `target_name` is the default value, 'Objective Value'. "
+            "This setting may confuse a plot."
```

Comment:
A minor. This last sentence might itself be a bit confusing. Also, it's that user that's confused and not the plot. I actually think that your warning message is good even without this last sentence.